twisted/test/test_defer.py

```diff
@@ -2357,7 +2357,7 @@ def test_waitUntilLockedWithTimeoutUnlocked(self):
         but the lock is unlocked before our timeout.
         """
         def onTimeout(f):
-            f.trap(defer.TimeoutError)
+            f.trap(error.TimeoutError)
```

Comment:
How do I test that?   That code is inside a test that is supposed to be testing a condition that is never supposed to happen.